demo.txt

```diff
@@ -1 +1,3 @@
-hello world !!!
\ No newline at end of file
+hello world !!!
+
+My name is Nathan.
```

Comment:
this is good.
Let's merge this. Next, let's put more information about your hobbies.